packages/remirror__extension-file/src/file-extension.tsx

```diff
@@ -88,7 +88,7 @@ export class FileExtension extends NodeExtension<FileOptions> {
         url: { default: '', validate: 'string' },
         fileName: { default: '', validate: 'string' },
         fileType: { default: '', validate: 'string' },
-        fileSize: { default: 0, validate: 'number' },
+        fileSize: { default: 0, validate: 'string|number' },
```

Comment:
Notice that in `parseDOM`, we simply use `const fileSize = anchor.getAttribute('data-filesize');` without converting the value into a number, that means existing `file` nodes having `fileSize` as `string`.